.github/workflows/cronjob.yml

```diff
@@ -1,13 +1,9 @@
 name: Cronjob
 
 # Run every hour on the hour.
-# `actions` is the name of my prototyping branch, so also run this job when I push there.
 on:
   schedule:
     - cron: "0 * * * *"
-  push:
-    branches:
-      - actions
```

Comment:
The actions branch is still magical, but now it only triggers the `make recheck` workflow (in another file)